compute/README.md

```diff
@@ -0,0 +1,55 @@
+---
+page_type: sample
+languages:
+- golang
+products:
+- azure
+description: "These code samples will show you how to manage Compute using Azure SDK for Golang."
+urlFragment: compute
+---
+
+# Getting started - Managing Compute using Azure Golang SDK
+
+These code samples will show you how to manage Compute using Azure SDK for Golang.
+
+## Features
+
+This project framework provides examples for the following services:
+
+### Compute
+* Using the Azure SDK for Golang - Compute Management Library [compute/armcompute](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/compute/armcompute) for the [Azure Compute API](https://docs.microsoft.com/en-us/rest/api/compute/)
```

Comment:
for those links, do we expect to manually provide them for other RPs?

Reply:
add to Resources section？